openedx/features/ucsd_support/views.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+import json
+
+from django.core.mail import send_mail
+from django.conf import settings
+from django.views.decorators.http import require_http_methods
+from django.http import HttpResponse
+from rest_framework import status
+
+email_template = '''
+    Course : {course}
+
+    {body}
```

Comment:
Please move this body variable to the end of this string.

Reply:
done